utils/storybook/stories.test.js

```diff
@@ -1,7 +1,7 @@
 /**
  * @jest-environment jsdom
  */
-import Adapter from 'enzyme-adapter-react-16';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
completely diff package from 16 needed to be installed

Reply:
Glad for this adapter! Curious how we might chart into R18?
First [google result](https://dev.to/wojtekmaj/enzyme-is-dead-now-what-ekl) I'm chewing on.